### PR TITLE
Ice cream and cones now contain proper reagents.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes the bug where ice cream cones did not actually contain any of the reagents advertised on the machine, nor did the machine itself. This has bugged me literally since the ice cream machine was added and now I can (sort of) into code and NOW IT IS FIXED.

Reagents are pretty self-explanatory: cocoa for chocolate, berry juice for strawberry, singulo drink for blue (this was how it was before; it's also in bluespace tomatoes for whatever it's worth). Waffle cones get 1 vitamin just like waffles do, chocolate cones instead get 1 cocoa (shit ain't healthy yo),

### CODE REVIEW

File: code/modules/food&drinks/kitchen machinery/icecream_vat.dm [hunk 1]
```diff
@@ -175,16 +184,28 @@
 	layer = 3.1
 	var/ice_creamed = 0
 	var/cone_type
+	var/flavour_type
```
Comment (RemieRichards) [inline code/modules/food&drinks/kitchen machinery/icecream_vat.dm]: this is always the same as ```flavour_name```, so why not just use ```flavour_name```?

Comment (bgobandit) [inline code/modules/food&drinks/kitchen machinery/icecream_vat.dm]: because of where the proc handles it

**Comment by Cheridan** [comment]
>nonstandardized europoor spelling

BOOOO
:us: :us: :us:

File: code/modules/food&drinks/kitchen machinery/icecream_vat.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/food&drinks/kitchen machinery/icecream_vat.dm]: ```flavour_name``` is passed to the only proc you use ```flavour_type``` in though...

Comment (bgobandit) [inline code/modules/food&drinks/kitchen machinery/icecream_vat.dm]: let me see if I can actually get it to work, it wasn't recognized when I tried

**Comment by optimumtact** [comment]
s cones

**Pull request merged**